content/en/blog/_posts/2020-01-17-Docs-Review-2019.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog
+title: "Reviewing 2019 in Docs"
+date: 2020-01-17
+slug: reviewing-2019-in-docs
+---
+
+**Author:** Zach Corleissen (Cloud Native Computing Foundation)
+
+Hi, folks! I'm one of the co-chairs for the Kubernetes documentation special interest group (SIG Docs). This blog post is a review of SIG Docs in 2019. Our contributors did amazing work last year, and I want to highlight their successes. 
+
+Although I review 2019 in this post, my goal is to point forward to 2020. I observe some trends in SIG Docs--some good, others troubling. I want to raise visibility before those challenges increase in severity.
+
+## The good
+
+There was much to celebrate in SIG Docs in 2019.
+
+Kubernetes docs started the year with three localizations in progress. By the end of the year, we ended with ten localizations available, four of which (Chinese, French, Japanese, Korean) are reasonably complete. The Korean and French teams deserve special mentions for their contributions to git best practices across all localizations (Korean team) and help bootstrapping other localizations (French team).
+
+Despite significant transition over the year, SIG Docs [improved its review velocity](https://k8s.devstats.cncf.io/d/44/pr-time-to-approve-and-merge?orgId=1&var-period=w&var-repogroup_name=SIG%20Docs&var-apichange=All&var-size_name=All&var-kind_name=All), with a median review time from PR open to merge of slightly greater than 24 hours. 
```

Comment:
```suggestion
Despite significant transition over the year, SIG Docs [improved its review velocity](https://k8s.devstats.cncf.io/d/44/pr-time-to-approve-and-merge?orgId=1&var-period=w&var-repogroup_name=SIG%20Docs&var-apichange=All&var-size_name=All&var-kind_name=All), with a median review time from PR open to merge of just over than 24 hours. 
```

Reply:
@celestehorgan Thanks for the review. Would you please say more about this edit? Also, nit about the nit... 😉 

> merge of **just over than** 24 hours.

Reply:
"slightly greater" just sounded awkward. (And yes, I caught your nit about the nit upon second read-through).

Take it or leave it, I'm ambivalent :)

Reply:
Done. 👍